packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1329,6 +1330,28 @@ export default class NewVm extends BaseComponent {
 
   // DISKS -----------------------------------------------------------------------
 
+  _getSrsNotOnSameHost = createSelector(
+    () => this.state.state.existingDisks,
+    () => this.state.state.VDIs,
+    () => this.props.srs,
+    (existingDisks, vdis, srs) => {
+      let container
+      const predicate = srId =>
+        container !== undefined
+          ? container !== srs[srId].$container
+          : ((container = get(() => srs[srId].$container)), false)
+
+      return (
+        vdis.some(({ SR: srId }) => srId !== null && predicate(srId)) &&
```

Comment:
In which case the Property `SR` can be null?

Reply:
if we clear the selected `SR`.

Reply:
True, i think is not the case for this issue. but IMHO, the SR should be required is the select. @pdonias what do you think?